Image icon link cards

milosyuki opened the pull request:
This PR includes Image icon link card component with 3 variants: `image_icon_link_card`, `icon_link_card` and `dapp` and is being showcased in the storybook as well. This is the component made of different components displayed in the figma as "Home page cards". The only one's missing are Large card and Community card which will be covered in the next PR.

Also, home card icons are created and added to the storybook.

milosyuki committed:
Icon image link card

robwalsh21 commented on _data/pages/en/developers.yml:
@milosyuki I'm noticing some rendering issues with the  cards under courses in dark mode  
![CleanShot 2023-04-14 at 15 22 47](https://user-images.githubusercontent.com/5764504/232071498-85ddf156-4fdb-42db-92e7-da5c440c3745.png)

milosyuki replied:
Thanks, fixed.

robwalsh21 commented on _data/pages/en/home.yml:
@milosyuki Another example of rendering issue with smaller card
![CleanShot 2023-04-14 at 15 26 30](https://user-images.githubusercontent.com/5764504/232072489-6280c350-a600-4ff9-869d-c48974dd81a9.png)

milosyuki replied:
Thanks, fixed.

milosyuki committed:
Merge branch 'dev' into cards-storybook

milosyuki committed:
Badges
Update Page “learn”
Adding community card to the storybook and fixing PR comments
Update Page “home”
Adding community card variant
Fixing large card image
Update Page “home”
Adding orientation to the large card
Update Page “home”
Adding large card to the config
Update Page “home”
Large card variant

robwalsh21 commented:
> This PR includes Image icon link card component with 3 variants: `image_icon_link_card`, `icon_link_card` and `dapp` and is being showcased in the storybook as well. This is the component made of different components displayed in the figma as "Home page cards". The only one's missing are Large card and Community card which will be covered in the next PR.
> 
> Also, home card icons are created and added to the storybook.

hey @milosyuki   Noticing some issues on hover with cards on home and developers page 
<img width="1365" alt="CleanShot 2023-04-18 at 10 58 12@2x" src="https://user-images.githubusercontent.com/5764504/232742708-ffa87b12-e36f-4b65-82f2-81f293a0a33d.png">
<img width="1320" alt="CleanShot 2023-04-18 at 10 57 54@2x" src="https://user-images.githubusercontent.com/5764504/232742741-66af5c54-df51-4c7d-af10-478a150bfc61.png">

milosyuki committed:
Merge branch 'dev' into cards-storybook

milosyuki committed:
Adding icon image link card to blocks

milosyuki committed:
Removing redundant

milosyuki committed:
Removing redundant

milosyuki committed:
Fixing LinkData import

milosyuki committed:
Removing redundant

milosyuki committed:
Fixing dark mode list card title

thecotne commented on src/components/Icons/CardIcons/DappsIcon/DappsIcon.tsx:
```diff
@@ -0,0 +1,54 @@
+import { CardIconProps } from "../IconInterface";
+
+const DappsIcon: React.FC<CardIconProps> = ({ gradientColor1, gradientColor2, filter, borderRadius }) => {
+  const gradientIdPrefix = `gradient-${Math.random().toString(36).substr(2, 9)}`;
```
use this https://react.dev/reference/react/useId

milosyuki replied:
Thanks, fixed!

thecotne commented on src/app/[locale]/tutorials/(components)/TutorialsPage.tsx:
```diff
@@ -302,7 +302,7 @@ type Tutorial = {
 function CustomHits() {
   const { hits, showMore, isLastPage } = useInfiniteHits<Tutorial>();
-
+console.log('hits ', hits)
```
this should be removed right?

milosyuki replied:
Thanks, removed.

thecotne commented on src/blocks/cards/IconLinkCard.tsx:
```diff
@@ -58,7 +58,7 @@ export const IconLinkCard = ({
             alignItems="flex-start"
           >
-            <Image src={icon} mb="20px" alt={title} />
+            {/* <Image src={icon} mb="20px" alt={title} /> */}
```
should this be removed instead of commenting it?

milosyuki replied:
Thanks, removed.

thecotne commented on src/components/Card/style.css:
```diff
@@ -0,0 +1,17 @@
+/*! @algolia/autocomplete-theme-classic 1.7.4 | MIT License | © Algolia, Inc. and contributors | https://github.com/algolia/autocomplete */
```
this comment should be removed

milosyuki replied:
Thanks, removed.

thecotne reviewed (changes requested):
looks good overall just few comments

milosyuki committed:
Fixing PR comments

thecotne reviewed (approved):
👍

robwalsh21 commented:
Did you get a chance to look at this @cxramina  ?

cxramina commented:
> Did you get a chance to look at this @cxramina ?

Yes, I send comments to Milos in Slack

robwalsh21 commented:
Great.  Thanks

milosyuki committed:
Updating grid card icons

milosyuki committed:
Merge branch 'dev' into cards-storybook

milosyuki committed:
Removing IconLinkCard
